ssh/rootfs/etc/profile.d/hassio.sh

```diff
@@ -1,4 +1,9 @@
-export PS1='\W $ '
+export PS1='\W'
+if [ "${USER-}" = root ]; then
```

Comment:
We are always root, not sure why this is needed in that case?

Reply:
We are always root, except when we aren't :) For example, after a `sudo -s -u hassio` for whatever reason. Granted, it's rare, but I don't see why we wouldn't just do the right thing here.

Reply:
🤷 sure, ok